src/frameworks/foundation/ns_thread.rs

```diff
@@ -13,6 +13,11 @@ pub const CLASSES: ClassExports = objc_classes! {
 
 @implementation NSThread: NSObject
 
++ (f64)threadPriority {
+    log!("TODO: [NSThread threadPriority] (not implemented yet)");
+    return 1.0 // always return 1.0
```

Comment:
Nitpick: in Rust, the final expression in a block (if not followed by a semicolon) is the return value, so the `return` is redundant and generally not used in this case. The comment is also a bit redundant.